test/libsolidity/SMTChecker.cpp

```diff
@@ -224,6 +224,36 @@ BOOST_AUTO_TEST_CASE(branches_clear_variables)
 		}
 	)";
 	CHECK_SUCCESS_NO_WARNINGS(text);
+	// Variable is reset in both branches
```

Comment:
This comment (and also the one below) is not correct, is it?

Reply:
What I meant is that program variable `a` is first assigned something (before the `if`), and then is reassigned inside both branches. I guess "reassigned" would be better than "reset" there.

Reply:
ah I see. Yeah, that's fine.